.circleci/config.yml

```diff
@@ -119,17 +118,17 @@ jobs:
             ./scripts/prepare-circle-integration-tests.sh
             export COMMIT="${CIRCLE_SHA1:8:8}"
             export CONTAINER_TAG="${COMMIT}"
-            export CONTAINER_FORCE_PULL="true"
+            # export CONTAINER_FORCE_PULL="true"
```

Comment:
Any reason to not simply delete this line?

Reply:
No reason, should be deleted.
Wanted to keep it here for reference but it's not a. good practice in any case